code/game/atoms.dm

```diff
@@ -464,4 +464,7 @@ its easier to just keep the beam vertical.
 	if((!message))
 		return
 	for(var/mob/O in hearers(src, null))
-		O.show_message("<span class='game say'><span class='name'>[src]</span> [atom_say_verb], \"[message]\"</span>",2)
\ No newline at end of file
+		O.show_message("<span class='game say'><span class='name'>[src]</span> [atom_say_verb], \"[message]\"</span>",2)
+
+/atom/proc/update_addictions(var/datum/reagent/R, var/amount)

```

Comment:
No offense, but uh..why in the world did you make a mob related proc---a proc for every single atom?

Aside from my disagreement with the addiction system, this should be under carbon, not every single atom.